apps/idam/idam-web-admin/ithc.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: idam-web-admin
+  namespace: idam
+spec:
+  releaseName: idam-web-admin
+  values:
+    java:
+      image: hmctspublic.azurecr.io/idam/web-admin:preview-7f70580-20220314172543
+      ingressHost: idam-web-admin.platform.hmcts.net
+      aadIdentityName: idam
+      useInterpodAntiAffinity: true
+      environment:
+        STRATEGIC_SERVICE_URL: https://idam-api.ithc.platform.hmcts.net
+        STRATEGIC_PUBLIC_URL: https://idam-web-public.ithc.platform.hmcts.net
+  chart:
```

Comment:
Can this be reviewed against the base idam-web-admin ? 
https://github.com/hmcts/cnp-flux-config/blob/master/apps/idam/idam-web-admin/idam-web-admin.yaml

Reply:
In ITHC web-admin needs to point to the ithc web-apps for IDAM, this configuration is correct.

Reply:
@nikola-naydenov-hmcts - sorry, was referencing the duplicated values in ithc.yam`l .spec.chart` `.spec.version` etc is duplicated from base

Reply:
Done